File: Python/Review/Q2/Calculator.py
while True:  # Do While (but not really xD)
    # (+) Addition
    # (-) Subtraction
    # (*) Multiplication
    # (/) Division
    try:
        n1 = int(input("Enter first number: "))
        n2 = int(input("Enter second number: "))
        OPERATOR = str(input("Enter operator: "))
        if (n1 == 999 and n2 == 999 and (OPERATOR == "C" or OPERATOR == "c")):
            print("Thank you for using the calculator!")
            break
        else:
            if OPERATOR == "+":
                print(n1 + n2)
            elif OPERATOR == "-":
                print(n1 - n2)
            elif OPERATOR == "*":
                print(n1 * n2)
            elif OPERATOR == "/":
                print(n1 / n2)
    except ZeroDivisionError:
        print("Cannot divide by zero!, You FOOL XD")

File: C/Review/Q2/Calculator.c
#include <stdio.h>

int main(void)
{
int n1, n2;
char operator;
    do
    {
        scanf("%d%d %c", &n1, &n2, &operator);
        if (operator == '+')
        {
            printf("%d\n", n1 + n2);
        }
        else if (operator== '-')
        {
            printf("%d\n", n1 - n2);
        }
        else if (operator== '*')
        {
            printf("%d\n", n1 * n2);
        }
        else if (operator== '/')
        {
            if (n2 == 0)
            {
                printf("Cannot divide by zero!, You FOOL XD");
                break;
            }
            printf("%d\n", n1 / n2);
        }
    } while ((n1 != 999 && n2 != 999 ) && operator != 'C');
        printf("Goodbye!\n");
    return 0;
}